{"name": "test_broken", "status": "broken", "statusDetails": {"message": "Exception: oops", "trace": "def test_broken():\n>       raise Exception('oops')\nE       Exception: oops\n\ntest_alluredemo1.py:33: Exception"}, "start": 1604324654681, "stop": 1604324654681, "uuid": "d2c79a4e-22a9-4643-86dd-7d96d1b91a9a", "historyId": "62e3d0efc8226834b589024da8762ee8", "testCaseId": "17acedbab6e0d57f20e780d7d6084ef4", "fullName": "test_alluredemo1#test_broken", "labels": [{"name": "suite", "value": "test_alluredemo1"}, {"name": "host", "value": "LAPTOP-UL20B91J"}, {"name": "thread", "value": "5676-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_alluredemo1"}]}